src/test/ui/unboxed-closures/non-tupled-call.stderr

```diff
@@ -0,0 +1,9 @@
+error[E0059]: cannot use call notation; the first type parameter for the function trait is neither a tuple nor unit
```

Comment:
It'd be nice if this suggested how to fix it, but that seems fine for another PR.

Reply:
I promise that I looked how to do this, and it's super hard, because we don't pass down the right spans here. Definitely can be a follow-up.